scripts/lemonbar/header_bar.sh

```diff
@@ -67,6 +67,19 @@ while :; do
     buf="${buf} |%{A:gsimplecal&:} ⮖ $(clock)%{A}"
 
     echo $buf
-    # use `nowplaying scroll` to get a scrolling output!
-    sleep 0.5 # The HUD will be updated every second
+}
+
+while true; do
+
+  read -t 0.25 new_wm_content
```

Comment:
@acmcarther One thing that is constantly a thorn in my side is how inconsistent the reported battery life is. I've been taking a look at it but most people are saying it's my battery that is bad, and not the battery file. However, the battery is basically new. Do you know what could cause this? If we could fix this it would be awesome.